src/Raven.Studio/typescript/models/database/dbGroup/databaseGroupGraph.ts

```diff
@@ -99,6 +99,8 @@ interface taskNodeWithCache extends taskNode {
 
 class databaseGroupGraph {
 
+    private static serverWideNamePrefixFromServer = "Server Wide Backup";
```

Comment:
Let's unify this and put this in one place. IIRC we have it in 2 places now. Too easy to desync.

Reply:
done